### ISSUE TITLE
Debugger breakpoint column missing

### ISSUE THREAD

**Issue by blackslate** [microsoft/vscode#128612]
Issue Type: <b>Bug</b>

![noBreakpointColumn](https://user-images.githubusercontent.com/12972520/125576950-ea9563a0-e98d-4165-a969-48df7dbc3ddc.png)

I recently made two changes in the Preferences:
1. Set Editor: Word Wrap Column to 70
2. Edited settings.json to include ```json
"editor.rulers": [
    70
  ],
```

Now the column for setting debugger breakpoints does not show. I do not know for certain whether the changes in the Preferences are related to the missing Breakpoints column or not.



I have tried:
* Undoing the Preferences changes that I made
* Opening a new window
* Restarting VS Code
* Installing the latest version of VS Code

None of these restore the debugger Breakpoints column

VS Code version: Code 1.58.1 (2aeda6b18e13c4f4f9edf6667158a6b8d408874b, 2021-07-13T06:20:02.397Z)
OS version: Linux x64 4.15.0-142-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz (4 x 2700)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 2|
|Memory (System)|7.64GB (0.23GB free)|
|Process Argv|--unity-launch --crash-reporter-id 5e013dff-6dfe-418c-acb1-e16c6736448c|
|Screen Reader|yes|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (31)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sublime-importer|auc|1.0.1
vagrant|bbe|0.5.0
markdown-emoji|bie|0.1.0
webvalidator|Cel|1.0.9
quick-html-previewer|dai|1.0.1
es7-react-js-snippets|dsz|3.1.1
permute-lines|ear|1.1.0
prettier-vscode|esb|8.0.1
shell-format|fox|7.1.0
vscode-pull-request-github|Git|0.28.0
vscode-graphql|Gra|0.3.16
HTMLHint|mka|0.10.0
vscode-docker|ms-|1.14.0
remote-containers|ms-|0.187.0
remote-ssh|ms-|0.65.7
remote-ssh-edit|ms-|0.65.7
remote-wsl|ms-|0.58.2
vscode-remote-extensionpack|ms-|0.21.0
sublime-keybindings|ms-|4.0.9
vsliveshare|ms-|1.0.4568
vsliveshare-pack|ms-|0.4.0
debugger-for-chrome|msj|4.12.12
printcode|nob|3.0.0
docthis|oou|0.8.2
shellman|Rem|5.6.0
LiveServer|rit|5.6.1
bash-debug|rog|0.3.9
markdown-preview-enhanced|shd|0.5.22
shellcheck|tim|0.14.4
pdf|tom|1.1.0
markdown-all-in-one|yzh|3.4.0


</details>
<!-- generated by issue reporter -->

**Comment by weinand** [comment]
@blackslate do you have the "editor.glyphMargin" setting set to "false"?

**Comment by blackslate** [comment]
Hello Andre!

On Wed, Jul 14, 2021 at 11:23 AM Andre Weinand ***@***.***>
wrote:

> @blackslate <https://github.com/blackslate> do you have the
> "editor.glyphMargin" setting set to "false"?
>
I did! But I do not remember setting that. And "glyphMargin" is not
something that I would have thought of looking for when my mind was full of
words like "debug" "breakpoint" and "checkpoint". But now I know what to
look for, I was able to find Editor: Glyph Margin by typing "debug" in the
Settings search field. Perhaps another name might have been a more obvious
choice.

Thanks for giving my breakpoints back : )

James
